src/Provider/WMTSProvider.js

```diff
@@ -79,11 +80,26 @@ function getXbilTexture(tile, layer, targetZoom) {
 
     return OGCWebServiceHelper.getXBilTextureByUrl(urld, layer.networkOptions).then((texture) => {
         texture.coords = coordWMTS;
+        let min;
+        let max;
+
+        // if pitch is full texture, the min and max are already known
+        if (pitch.z == 1.0 && pitch.w == 1.0) {
+            min = !texture.min ? 0 : texture.min;
```

Comment:
Is this test `!texture.min` to handle the:
```js
    if (max === -1000000 || min === 1000000) {
        return { min: null, max: null };
    }
```
case from XbilParser?

If yes, then would it make more sense to adapt XbilParser to return `{ max: 0, min: 0 }` in this case instead of fixing here?

Reply:
yes, it's changed